src/wallet/components/WalletDropdownBaseName.tsx

```diff
@@ -0,0 +1,58 @@
+import { base } from 'viem/chains';
+import { useAccount } from 'wagmi';
+import { useName } from '../../identity/hooks/useName';
+import { Spinner } from '../../internal/components/Spinner';
+import { baseNameSvg } from '../../internal/svg/baseNameSvg';
+import { cn, pressable, text } from '../../styles/theme';
+import type { WalletDropdownBaseNameReact } from '../types';
+
+export function WalletDropdownBaseName({
+  className,
```

Comment:
There are some extra customization we might want to add here as follow up PR. I let you think on this.

Reply:
Will do 👍